spec/acceptance/ntp_daemon_opts_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper_acceptance'
+
+config = if fact('osfamily') == 'redhat'
+           '/etc/sysconfig/ntp'
+         else
+           '/etc/default/ntp'
+         end
+
+describe 'ntp class with daemon options:', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+  context 'when successful' do
+    let(:pp) { "class { 'ntp': user => 'ntp', daemon_extra_opts => '-g -i /var/lib/ntp' }" }
+
+    it 'runs twice' do
+      2.times do
+        apply_manifest(pp, catch_failures: true) do |r|
+          expect(r.stderr).not_to match(%r{error}i)
+        end
+      end
+    end
+  end
+
+  describe file(config.to_s) do
```

Comment:
Test's are failing here. Are you sure the file path is right?

Reply:
it seems there is issue with osfamily identification.
config is mapped to debian value instead of redhat
https://travis-ci.org/juju4/puppetlabs-ntp/jobs/507636546#L3441

Reply:
Ah, I see where the error is coming from, facter returns OS Family with uppercase character's. Easy mistake to make.
A good fix would be to swap it to a ServerSpec call as we have recently done in the rest of the test code, similar what I've shown below:
```
config = if os[:family] == 'redhat'
           '/etc/sysconfig/ntp'
         else
           '/etc/default/ntp'
         end
```
ServerSpec retains the information that it retrieves so that it only need's to actually retrieve the OS Family once. As a note in this case it is supposed to be lowercase.

Reply:
Also this test will need some code to account for `suse` family OS. Sorry if I'm being nitpicky.

Reply:
you will have too help me how to do this on suse. not familiar with it :)
and quick search gave only client configuration info, not where daemon/user options are supposed to be done.
but first, fixing centos & debian

Reply:
Ok, have to make a correction to what I said earlier, apparently os[:family] returns to OS type, so ubuntu instead of debian.
Instead please use `host_inventory['facter']['os']['family']` which will return the os family with a capital letter at the start. Sorry for giving you the wrong information before.

Reply:
ServerSpec is a new thing for me sorry

Reply:
Also, shouldn't `'/etc/sysconfig/ntp'` be `'/etc/sysconfig/ntpd'`?